### ISSUE TITLE
gatling-recorder (maven) not converting har file

### ISSUE THREAD

**Issue by anemer** [gatling/gatling#3805]
Gatling recorder, from Maven Gatling plugin, is not converting HAR file to simulation, without any error message.

Environment: 
* OS : windows 10 
* Maven gatling-maven-plugin:3.0.1 
* Gatling 3.2.1

Tested on our app, and also on https://gatling.io/blog
* Example har file : [gatling_blog.har.zip](https://github.com/gatling/gatling/files/3716941/gatling_blog.har.zip)
* Gatling recorder configuration: ![2019-10-11 11_56_16-Gatling Recorder - Configuration](https://user-images.githubusercontent.com/15840278/66643974-28417d80-ec20-11e9-9fca-973835e50b78.png)

->no reaction to click on _Start button_. No popup, no error log.

**Comment by anemer** [comment]
Ok. For info, it works via the bundle (gatling-charts-highcharts-bundle-3.2.1) :
`recorder.bat -m=Har -hf="C:\dev\test\gatling_blog.har" -sf="c:\dev\test"` command gives the good result

**Comment by slandelle** [comment]
Are you sure you don't have any exception?

**Comment by anemer** [comment]
You are right, at click on the _start_ button, we have:

```
Exception in thread "AWT-EventQueue-0" java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.ObjectMapper.readerFor(Ljava/lang/Class;)Lcom/fasterxml/jackson/databind/ObjectReader;
	at org.json4s.jackson.JsonMethods.parse(JsonMethods.scala:20)
	at org.json4s.jackson.JsonMethods.parse$(JsonMethods.scala:19)
	at org.json4s.jackson.JsonMethods$.parse(JsonMethods.scala:55)
	at io.gatling.recorder.har.HarParser$.parseHarEntries(HarParser.scala:51)
	at io.gatling.recorder.har.HarReader$.readStream(HarReader.scala:45)
	at io.gatling.recorder.har.HarReader$.$anonfun$readFile$1(HarReader.scala:42)
	at io.gatling.commons.util.Io$.withCloseable(Io.scala:115)
	at io.gatling.recorder.har.HarReader$.readFile(HarReader.scala:42)
	at io.gatling.recorder.scenario.ScenarioExporter$.$anonfun$exportScenario$2(ScenarioExporter.scala:73)
	at io.gatling.commons.validation.package$.safely(package.scala:34)
	at io.gatling.recorder.scenario.ScenarioExporter$.exportScenario(ScenarioExporter.scala:72)
	at io.gatling.recorder.controller.RecorderController.startRecording(RecorderController.scala:60)
	at io.gatling.recorder.ui.RecorderFrontEnd.startRecording(RecorderFrontEnd.scala:64)
	at io.gatling.recorder.ui.swing.frame.ConfigurationFrame.reloadConfigurationAndStart(ConfigurationFrame.scala:616)
	at io.gatling.recorder.ui.swing.frame.ConfigurationFrame.$anonfun$start$1(ConfigurationFrame.scala:112)
	at scala.swing.Action$$anon$1.apply(Action.scala:63)
	at scala.swing.Action$$anon$2.actionPerformed(Action.scala:81)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
	at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
	at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:252)
	at java.awt.Component.processMouseEvent(Component.java:6533)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6298)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)


```

**Comment by slandelle** [comment]
You seem to be messing up with Gatling's dependencies and forcing downgrade.

The `ObjectMapper.readerFor(Class)` method was introduced in Jackson 2.6 (released in 2015), Gatling 3.2.1 pulls 2.9.9.3.
